lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -752,33 +752,12 @@ fromAlonzoPParams eraInfo currentNodeProtocolParameters pp =
 
 -- | Extract the current network decentralization level from the given set of
 -- protocol parameters.
---
--- According to the Design Specification for Delegation and Incentives in
--- Cardano, the decentralization parameter __/d/__ is a value in the range
--- '[0, 1]', where:
---
---   * __/d/__ = '1' indicates that the network is /completely federalized/.
---   * __/d/__ = '0' indicates that the network is /completely decentralized/.
```

Comment:
Hi @HeinrichApfelmus 

I feel this comment was actually quite informative. Perhaps there's a way of retaining just this part to serve as documentation? (Perhaps this could be at home within in a `Primitive.Types.DecentralizationLevel` module.)

Reply:
This particular comment is actually a *copy* of the Haddock comment associated with the `DecentralizationLevel` type — consistency is improved by removing the copy. The original is found (and preserved) here:

https://github.com/input-output-hk/cardano-wallet/blob/0c58019050a4906aa83e989fdb8c81a3f98e6f78/lib/core/src/Cardano/Wallet/Primitive/Types.hs#L1210-L1227

Reply:
Ah thanks for pointing this out @HeinrichApfelmus, I hadn't spotted that!